Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
But what about `vcbuild.bat` and the case of  #12830 & #12771
I was actually thinking of refactoring _ALL_ the similar cases into `tools/make` to help alleviate #12425
But on the other hand `Makefile` can almost be read like a configuration file...

Anyway we should all move to `ninja` with one `node.ninja` for all ✊